src-electron/generator/template-util.js

```diff
@@ -197,6 +204,20 @@ async function ensureZclPackageIds(context) {
   }
 }
 
+async function ensureTemplatePackageCategory(context) {
```

Comment:
Missed some comments for the new function

Reply:
I'll add them....